plugins/jobtype/src/azkaban/jobtype/connectors/TdchConstants.java

```diff
@@ -0,0 +1,27 @@
+package azkaban.jobtype.connectors;
+
+public interface TdchConstants {
+  public static final String MAP_REDUCE_PARAMS =
```

Comment:
MAP_REDUCE_PARAMS seems like a configuration, should we allow flexibility ?
Can we move it to private.properties (only azkaban admin can control) or plugin.properties (user will be able to override) ?